src/jquery.sidebar.js

```diff
@@ -114,7 +112,15 @@
          * */
         self.on("sidebar:close", function(callback) {
             var properties = {};
-            properties[settings.side] = settings.range[0];
+            if (settings.side == "left" || settings.side == "right")
```

Comment:
feel free to change this. I don't know how to update my git pull request. maybe you can do it on your end?